internal/dag/status_test.go

```diff
@@ -2623,6 +2623,64 @@ func TestDAGStatus(t *testing.T) {
 				Valid(),
 		},
 	})
+
+	noRouteAction := &contour_api_v1.HTTPProxy{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "roots",
+			Name:      "noRouteAction",
+		},
+		Spec: contour_api_v1.HTTPProxySpec{
+			VirtualHost: &contour_api_v1.VirtualHost{
+				Fqdn: "example.com",
+			},
+			Routes: []contour_api_v1.Route{{
+				Conditions: []contour_api_v1.MatchCondition{{
+					Prefix: "/foo",
+				}},
+			}},
+		},
+	}
+	run(t, "No routeAction specified is invalid", testcase{
+		objs: []interface{}{noRouteAction},
+		want: map[types.NamespacedName]contour_api_v1.DetailedCondition{
+			{Name: noRouteAction.Name, Namespace: noRouteAction.Namespace}: fixture.NewValidCondition().
+				WithError(contour_api_v1.ConditionTypeRouteError, "routeActionCountNotValid",
+					"must set exactly one of route.services or route.requestRedirectPolicy or route.directResponsePolicy"),
+		},
+	})
```

Comment:
This test case looks identical to "invalid HTTPProxy due to empty route.service", let's drop this new one and just revise the existing one's name to indicate that it covers no route actions.